react-native/ios/RealmReact.xcodeproj/project.pbxproj

```diff
@@ -120,9 +120,9 @@
 		F60690241CA277CC0003FB26 /* Products */ = {
 			isa = PBXGroup;
 			children = (
-				F606902D1CA277CD0003FB26 /* libGCDWebServers.a */,
+				F606902D1CA277CD0003FB26 /* liblibGCDWebServers.a */,
```

Comment:
Don't think that is used - I renamed it and changed it back so think it is just an artifact - will remove.